---
title: "dataproc/WARN/2022_001"
linkTitle: "WARN/2022_001"
weight: 1
type: docs
description: >
  Dataproc clusters are not failed to stop due to the local SSDs
---

**Product**: [Cloud Dataproc](https://cloud.google.com/dataproc)\
**Rule class**: WARN - Something that is possibly wrong

### Description

You cannot stop clusters with local SSDs attached since it triggers shutdown to
the VM. However, if you do shut down a VM using local SSDs, then you can't
start the VM again later, and the data on the local SSD is lost.

### Remediation

Make sure that you migrate your critical data from the local SSD to a
persistent disk or to another VM before stopping the Dataproc clusters.

### Further information

- [Starting and stopping clusters](https://cloud.google.com/dataproc/docs/guides/dataproc-start-stop#limitations)
- [Stop and start a VM](https://cloud.google.com/compute/docs/instances/stop-start-instance#restrictions)
